eng/pipelines/templates/steps/compilation.check.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  - name: Artifact
+    type: object
+    default: {}
+  - name: ArtifactName
+    type: string
+    default: "not-specified"
+  - name: RetryLimit
+    type: number
+    default: 30
+
+steps:
+  - download: current
+    artifact: ${{parameters.ArtifactName}}-signed
+  - pwsh: |
+      $pomf = Get-ChildItem "$(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{parameters.Artifact.groupId}}/${{parameters.Artifact.name}}/*.pom"
+      while ($retries++ -lt ${{parameters.RetryLimit}}) {
+        Write-Host "mvn --no-transfer-progress -f $pomf compile"
```

Comment:
We were doing a "install" now we are doing a "compile" is that correct?

Reply:
Yup that's correct